azkaban-common/src/main/java/azkaban/imagemgmt/daos/RampRuleDao.java

```diff
@@ -0,0 +1,23 @@
+package azkaban.imagemgmt.daos;
+
+import azkaban.imagemgmt.models.ImageRampRule;
+
+
+/**
+* Data access object (DAO) for accessing image ramp rule that deny an image ramping version for flows.
+* This interface defines add/remove/modify an image ramp rule.
+* */
+public interface RampRuleDao {
+
+  boolean isHPRule(String ruleId);
```

Comment:
Add comments for each method here?
Specially the full name of "HP".

Reply:
Also add "throws ImageMgmtException" if the method will throw. Refer to ImageRampupDao.java.

Reply:
Done and I would add more details related to HP flow rule in CreateHPFlow API implementation. This is only a marker for them.